src/DistributionApi.php

```diff
@@ -112,18 +119,60 @@ public function isModuleOnDisk(string $moduleName): bool
         return $this->moduleDataProvider->isOnDisk($moduleName);
     }
 
+    /**
+     * Extracts the download URL from a module data structure
+     *
+     * @param array{download_url?: string} $module Module data structure, from API response
+     *
+     * @return string Download URL
+     */
+    protected function getModuleDownloadUrl(array $module): string
+    {
+        if (!isset($module['download_url'])) {
+            throw new RuntimeException('Could not determine URL to download the module from');
+        }
+
+        return $this->addShopInfoToUrl($module['download_url']);
+    }
+
+    /**
+     * Returns the URL to the list of modules for this version
+     *
+     * @return string
+     */
+    private function getModulesListUrl(): string
+    {
+        $url = self::API_ENDPOINT . '/modules/' . $this->prestashopVersion;
+
+        return ((bool) getenv(self::URL_TRACKING_ENV_NAME) === true) ? $this->addShopInfoToUrl($url) : $url;
```

Comment:
I think it should be the opposite, if it exists and it equals `false`, then we don't send an URL. Right?

Reply:
You're right ! thank you, I juste updated the code